### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by jackkoenig**
* chisel: 5.0.0 => 5.1.0
* chiseltest: 5.0.0 => 5.0.2
* scala: 2.13.8 => 2.13.12
* sbt: 1.8.0 => 1.9.7

Also remove -P:chiselplugin:genBundleElements, it has been deprecated for a while.

### CODE REVIEW

File: build.sc [hunk 1]
```diff
@@ -7,25 +7,24 @@ import scalalib._
 // support BSP
 import mill.bsp._
 
-object %NAME% extends SbtModule { m =>
+object foo extends SbtModule { m =>
```
Comment (jiegec) [inline build.sc:10]: Is this change intended?

Comment (jackkoenig) [reply build.sc:10]: No it was not, whoops, I made that change for local testing.

Thanks for pointing it out, will fix.

Comment (jackkoenig) [reply build.sc:10]: Fixed, thanks @jiegec!
